### PR TITLE
Use DRF 3.1 `pagination_class`

### LINKED ISSUE #3251: Import UI not showing all organizations
I suspect I recently went over 10 organizations in my GitHub account. Now when I see my list of organizations, I'm missing an organization for which I'm an owner (/yougov):

image

I suspect there's either some pagination going on with the organizations or there's a hard-coded limit of 10. In either case, I seem to be unable to filter by that organization.

### PR DESCRIPTION
**Pull request by humitos**
Closes #3251

### CODE REVIEW

**Comment by humitos** [comment]
@ericholscher @agjohnson since we were talking that mixing style and logic PR is kind of tedious, I created another PR based on this where I ran `pre-commit` on the modified files and tweaked a little manually.

Let me know if you like this approach. It a little tedious to make two different PR but I hope it simplies the review.

**Comment by safwanrahman** [comment]
I think, rather than fixing from the backend, this should be controlled by `page_size` parameter from the frontend. what do you think @humitos ?

**Comment by humitos** [comment]
@safwanrahman good catch!

I think it's a good idea. I'd like to fix the bug and make it work as it was working before since it could be other places that use that endpoint and may rely on that pagination size also.

From #3251 

> I think there should be a button in the UI for going next

Yes, we should think in a non-intrusive UI design for this. Most of the users won't have more than 25 organizations (in fact we have just one report of a person having 11 :) ).

But anyway, I agree with both suggestions but I'd like to make them in different PR.

If you agree with me, would you mind openning these two issues? Thanks!

File: readthedocs/rtd_tests/tests/test_api.py [hunk 1]
```diff
@@ -265,6 +265,43 @@ def test_project_features_multiple_projects(self):
         self.assertIn('features', resp.data)
         self.assertEqual(resp.data['features'], [feature.feature_id])
 
+    def test_project_pagination(self):
+        for _ in range(100):
+            get(Project)
+
+        resp = self.client.get('/api/v2/project/')
```
Comment (safwanrahman) [inline readthedocs/rtd_tests/tests/test_api.py]: I think you can get the url by using `reverse`. So it will be easier to manage in the future

**Pull request merged**